scripts/build/TestImpactAnalysis/Testing/test_tiaf_unit_tests.py

```diff
@@ -0,0 +1,233 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+from logging import getLogger
+import os
+import json
+from typing import Counter
+from tiaf import TestImpact
+from tiaf_driver import main
+import pytest
+
+logging = getLogger("tiaf")
+
+
+def assert_list_content_equal(list1, list2):
+    assert Counter(list1) == Counter(list2)
+
+
+class TestTIAFUnitTests():
+
+    def test_config_files_exist(self, config_path):
+        # given:
+        # config path, and files have been built
+        # when:
+        # we look for the file
+        # then:
+        assert os.path.exists(config_path)
+
+    def test_valid_config(self, caplog, tiaf_args, mock_runtime, mocker, default_runtime_args):
```

Comment:
I suggest using Osherove convention for unit test naming. It helps maintainers and reviewers understand more about the test at a glance
https://osherove.com/blog/2012/5/15/test-naming-conventions-with-unit-of-work.html

Reply:
Will implement this.